config/websocket.js

```diff
@@ -0,0 +1,82 @@
+
+import jwt from 'jsonwebtoken';
+import config from './config';
+import notificationCtrl from '../server/controllers/notification.controller';
+
+let server;
+let sockets = [];
+
+function setServer(io) {
+  server = io;
+  server.on('connection', (socket) => {
+    socketEvents(socket);
+  });
+}
+
+function askRegister(socket) {
+  socket.emit('app.register');
+}
+
+function registerSocket(socket, token) {
+  try {
+    const data = jwt.verify(token, config.jwtSecret);
+    if (!data.username) return false;
```

Comment:
Similar to my comment on the single line function, I think these should be in a block format, but again, your PR is fine to merge.

```
if (!data.username) {
  return false;
}
```

Reply:
This I think the opposite. I find it easier and simpler to use a single line, single sentence. But we are practically talking about taste here

Reply:
Totally